docs/generator.py

```diff
@@ -420,3 +441,68 @@ def split_operation_names(text):
     replace_regexp = r'\1 \2'
     updated_text = re.sub('(.)([A-Z][a-z]+)', replace_regexp, text)
     return re.sub('([a-z0-9])([A-Z])', replace_regexp, updated_text)
+
+
+#####
```

Comment:
Please, move these functions to a separate file (maybe, `filter.py` or `jinja_filter.py`) and document them.